Sources/ScipioKit/Producer/PIF/FrameworkBundleAssembler.swift

```diff
@@ -111,16 +110,13 @@ struct FrameworkBundleAssembler {
 
         if !subdirectoryComponents.isEmpty {
             try fileSystem.createDirectory(
-                headerDir.appending(components: subdirectoryComponents).asURL,
+                headerDir.appending(components: subdirectoryComponents),
                 recursive: true
             )
         }
         try fileSystem.copy(
-            from: header.asURL,
-            to: headerDir
-                .appending(components: subdirectoryComponents)
-                .appending(component: header.basename)
-                .asURL
+            from: header,
+            to: headerDir.appending(components: subdirectoryComponents)
```

Comment:
ref:
- https://github.com/giginet/Scipio/pull/227#discussion_r2184473653
- 5cdaecd399c9b4062e336bee5dc6ec98e4d34b45

`.appending(component: header.basename)` was wrongly deleted. I just suggested to remove an unnecessary empty line.